include/uv/win.h

```diff
@@ -265,19 +265,14 @@ typedef union {
 
 typedef union {
   struct {
```

Comment:
```suggestion
typedef struct {
```
Can you remove the extra `state_` indirection?

In v2, I will make this `typedef SRWLOCK uv_rwlock_t`

Reply:
Done :)